kernel_tracing/metric_evaluation/readme.md

```diff
@@ -0,0 +1,34 @@
+# Metric Evaluation
+
+The `test.py` script in this directory executes tests specified in a JSON file.
+The JSON file is formatted as such:
+
+```json
+{
+  "cases": [
+    {
+      "name": "Name to be displayed in output",
+      "bpftrace": "path to the bpftrace script used to collect metrics, relative to spec file",
+      "shell": "shell command to be run alongside bpftrace script, relative to spec file",
+      "conditions": [
+        "list of conditions that will be evaluated against bpftrace output",
+        "these are python expressions",
+        "bpftrace vars can be referenced with @{var name}, ie @avg",
+        "maps are referenced by @{var name}[{key name(s)}], ie @avg[ipc_time]"
```

Comment:
so in the example above, that does '@val' refer to?

Reply:
That refers to the static val = 42 assignment in the no_dependencies script. I'll be renaming it to test_value so it's more clear.